plugins/extractors/mysql/mysql_test.go

```diff
@@ -38,16 +41,16 @@ func TestMain(m *testing.M) {
 		Env: []string{
 			"MYSQL_ALLOW_EMPTY_PASSWORD=true",
 		},
-		ExposedPorts: []string{"3306"},
+		ExposedPorts: []string{"3306", port},
 		PortBindings: map[docker.Port][]docker.PortBinding{
 			"3306": {
```

Comment:
should we update this map key as well?

Reply:
No, key is the one that needs the default port and it maps it to the `port` we provide.